dicegame2/script.js

```diff
@@ -0,0 +1,89 @@
+//variables list
+const restart = document.querySelector("#restart");
+const roll = document.querySelector("#roll");
+const hold = document.querySelector("#hold");
+const player1 = document.querySelector("#player1");
+const player2 = document.querySelector("#player2");
+const score1 = document.querySelector("#score1");
+const score2 = document.querySelector("#score2");
+let cScore1 = document.querySelector("#currentscore1");
+let cScore2 = document.querySelector("#currentscore2");
+const box1 = document.querySelector("#box1");
+const box2 = document.querySelector("#box2");
+let rollScore1 = 0;
+let rollScore2 = 0;
+let won = false;
+let activeplayer = 1;
+
+//begin by clicking new game, show roll dice and hold buttons
+restart.addEventListener("click", (event) => {
+    roll.style.display = "block"; //display roll button
+    hold.style.display = "block"; //display hold button
+    if (won == true) { //if someone has won and new game is clicked, the page refreshes
+        location.reload();}
+});
+
+//roll button
+roll.addEventListener("click", (event) => {
+    let diceRoll = Math.floor(Math.random()*6) + 1; //diceroll number generates
+    let diceNum = document.getElementById("diceImg").src = "./img/dice" + diceRoll + ".png";
+        diceNum = document.querySelector("#diceImg"); //dice image shows based on diceroll
+        diceNum.style.display = "block"; 
+
+    if (activeplayer == 1) {  //active player select
+        if (diceRoll == 1) {  //has a 1 been rolled?
+            activeplayer = 2; //if 1 is rolled, swap to player2
+            cScore1.textContent = 0; //clear player1 current score
+            reset()
+            return
```

Comment:
Since you're not returning an explicit value (bare return gives you 'undefined') you can combine these lines together `return reset()` you may have to change it later if you suddenly do want to return an explicity value, but for concise code, you can do that.